aspnetcore/includes/bind-get.md

```diff
@@ -3,3 +3,4 @@
 >
 > To bind a property on `GET` requests, set the [[BindProperty]](/dotnet/api/microsoft.aspnetcore.mvc.bindpropertyattribute) attribute's `SupportsGet` property to `true`:
 > `[BindProperty(SupportsGet = true)]`
+> For more information, see [YouTube, bind on GET](https://www.youtube.com/watch?v=p7iHB9V-KVU&feature=youtu.be&t=54m27s)
```

Comment:
Hum .... this is strange (I mean the prior lines). Recommend this (it's a cut-'n-paste for Lines 4-6 if you want it) ...

> To bind a property on \`GET\` requests, set the \[[BindProperty]](xref:Microsoft.AspNetCore.Mvc.BindPropertyAttribute) attribute's \`SupportsGet\` property to \`true\`:
>
> \`\`\`csharp
> \[BindProperty(SupportsGet = true)]
> \`\`\`
>
> For more information, see \[ASP.NET Core Community Standup: Bind on GET discussion (YouTube)](https://www.youtube.com/watch?v=p7iHB9V-KVU&feature=youtu.be&t=54m27s).